ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1229,19 +1229,19 @@ func (r *RayClusterReconciler) calculateStatus(ctx context.Context, instance *ra
 }
 
 // Best effort to obtain the ip of the head node.
-func (r *RayClusterReconciler) getHeadPodIP(ctx context.Context, instance *rayv1.RayCluster) (string, error) {
+func (r *RayClusterReconciler) getHeadPodIPAndName(ctx context.Context, instance *rayv1.RayCluster) (string, string, error) {
```

Comment:
Maybe we can use `getHeadPod` which returns the head Pod and an err instead

Reply:
Hi @kevin85421, do you mean remove this function? We use this function to get the head IP and name and store them in the `HeadInfo`. It is actually this function that powers the `getHeadPod`.

Reply:
Discussed offline: we moved the `getHeadPodIPAndName` into `common.GetRayClusterHeadPod`.